packages/ignite-cli/src/commands/plugin.js

```diff
@@ -85,11 +85,13 @@ const createNewPlugin = async (context) => {
     (answers.template === 'Yes') &&
       { template: 'plugin/templates/Example.js.ejs', target: `${pluginName}/templates/${name}Example.js.ejs` },
     (answers.command === 'Yes') &&
-      { template: 'plugin/commands/example.js.ejs', target: `${pluginName}/commands/example.js` }
+      { template: 'plugin/commands/thing.js.ejs', target: `${pluginName}/commands/thing.js` },
```

Comment:
So `example.js` as a command name was confusing. I changed it to `thing.js`, as in `ignite generate thing MyThing`.